README.md

```diff
@@ -223,6 +224,56 @@ ChatDeleteResponse deleteResponse = slack.methods().chatDelete(
 assertThat(deleteResponse.isOk(), is(true));
 ```
 
+##### Open a dialog modal
+
+```java
+String token = "api-token";
+    
+    // Required.  See https://api.slack.com/dialogs#implementation
```

Comment:
You don't need to have indent from this line.

Reply:
Done.  Commit: https://github.com/seratch/jslack/pull/33/commits/823f7197a57defaae3cd4d12472a0f5321e05aa4